Rubberduck.Refactorings/Rename/RenameRefactoring.cs

```diff
@@ -64,6 +64,13 @@ public RenameRefactoring(IVBE vbe, IRefactoringPresenterFactory<IRenamePresenter
             _neverRenameIdentifiers = NeverRenameList();
         }
 
+        private RenameModel InitializeModel(Declaration target)
```

Comment:
The naming here seems off. This isn't "Initializing" the model, it's creating it.  How about something more like `CreateModelForDeclaration`?

Reply:
Sure, will rename.